content/body/input-mask.php

```diff
@@ -267,46 +267,38 @@ class="enable-table enable-table--centered-data enable-table--with-borders">
 
     <h3>Mouse UX</h3>
-    <p>For mouse users, when the click on what they think is the input field, they are actually clicking on the facade
-        stacked on top. JavaScript figures out where in the input data they are clicking and ensure the cursor in the
-        input field stacked underneath is placed in the right area. Because all mouse events are basically passed on to
-        the input field underneath, the user can select text with a mouse and the appropriate text is selected in the
-        input field so that is updated correctly. </p>
-
-    <h3>Screen-reader UX</h3>
-    <p>If the user stops typing for a while, the "formatted value" of the input field is announced (i.e. the input
-        field's value announced character by character). This is done via an ARIA live region which is described in the
-        code walkthrough above. So, instead of the screen reader reading the input field as a large integer (in this
-        case "two billion one hundred twenty three million one hundred twenty one thousand two hundred thirty one"), it
-        will read it as the phone number one digit at a time (i.e. two one two three one two one two three one). This
-        makes it easy for screen reader users to know what they just typed in.</p>
+    <p>For mouse users, when they click on what they think is the input field, they are actually clicking on the facade stacked on top. 
+        JavaScript figures out where in the input data they are clicking and ensures the cursor in the input field stacked underneath is placed in the right area. 
+        Because all mouse events are basically passed on to the input field underneath, the user can select text with a mouse, and the appropriate text is selected in the input field, so that is updated correctly.</p>
+
+    <h3>Screen Reader UX</h3>
+    <p>If the user stops typing for a while, the "formatted value" of the input field is announced (i.e., the input field's value is announced character by character). 
+        This is done via an ARIA live region, which is described in the code walkthrough above. 
+        So, instead of the screen reader reading the input field as a large integer (in this case, "two billion one hundred twenty-three million one hundred twenty-one thousand two hundred thirty-one"), 
+        it will read it as the phone number one digit at a time (i.e., two one two three one two one two three one). This makes it easy for screen reader users to know what they just typed in.</p>
 </details>
 
 
 <h2>Example 2: Dynamic Masking of Credit Card Fields</h2>
 
 
 <p>
-    Credit card fields are a little different. At the time of this writing, the format characters (i.e. the spaces) are
-    put into different places depending if it's an American Express (a.k.a. AMEX) Card or another credit card type:
+    The credit card fields are a little different. At the time of this writing, the format characters (i.e., the spaces) are
```

Comment:
I don't think we usually included commas after "i.e.". Do we need it here?
```suggestion
    The credit card fields are a little different. At the time of this writing, the format characters (i.e. the spaces) are
```